src/components/DatePicker/DatePicker.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+export default function DateRange() {
```

Comment:
- [ ]  Please, kindly refactor this function ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪